pom.xml

```diff
@@ -19,12 +19,14 @@
     </properties>
 
     <dependencies>
+        <!-- https://mvnrepository.com/artifact/junit/junit -->
```

Comment:
Just a little advice - this information is redundant and you can replace it with empty space for better readability